webapp/src/utils/validation/string.ts

```diff
@@ -13,13 +13,6 @@
  */
 
 import { t } from "i18next";
-import * as R from "ramda";
-
-type ValidationResult = string | true;
```

Comment:
Keep `ValidationResult` in a types.ts file, because it used by all validation functions